File: app/src/main/java/net/todd/mavelheroes/MarvelCharacter.java
package net.todd.mavelheroes;

import com.google.gson.annotations.SerializedName;

public class MarvelCharacter {
    @SerializedName("id")
    String id;
    @SerializedName("name")
    String name;
    @SerializedName("description")
    String description;

    public MarvelCharacter(String id, String name, String description) {
        this.id = id;
        this.name = name;
        this.description = description;
    }

    public String getName() {
        return name;
    }

    public String getBio() {
        return description;
    }
}

File: app/src/main/java/net/todd/mavelheroes/MarvelCharacter.kt
package net.todd.mavelheroes

import com.google.gson.annotations.SerializedName

class MarvelCharacter(@SerializedName("id")
                      internal var id: String, name: String, description: String) {
    @SerializedName("name")
    var name: String
        internal set
    @SerializedName("description")
    var bio: String
        internal set
    @SerializedName("thumbnail")
    internal var thumbnail: CharacterThumbnail? = null

    init {
        this.name = name
        this.bio = description
    }

    val imagePath: String
        get() = thumbnail!!.imagePath
}